.github/workflows/move-work-in-progress-prs.yml

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: Move PR To Work In Progress
+on:
+  issue_comment:
+    types: [created]
+    branch:
+      - main
+
+jobs:
+  move-pr-to-wip:
+    if: github.event.comment.body == '/pr mark wip' && github.event.issue.pull_request
+    runs-on: ubuntu-latest
+    steps:
+      - name: Move To Work In Progress
+        uses: actions/github-script@v6
+        with:
+          script: |
+            // Find "Code Reviews" project manually by name matching
+            // This avoids hardcoding the project ID
+            const projects = await github.rest.projects.listForRepo({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+            })
+            const code_reviews = projects.data.find(project => {
+              return project.name === 'Code Reviews';
+            })
+            if (!code_reviews) {
+              console.error("'Code Reviews' project not found!")
+              return
+            }
+
+            // Find "Work In Progress" column manually by name matching
+            // Also find the card of the PR in either "Initial Review" or "Final Review"
+            // This avoids hardcoding the column ID and card ID
+            const columns = await github.rest.projects.listColumns({
+              project_id: code_reviews.id,
+            });
+
+            const work_in_progress = columns.data.find(column => column.name === 'Work In Progress')
+            if (!work_in_progress) {
+              console.error("'Work In Progress' column not found!")
+              return
+            }
+
+            columns.data.forEach(column => {
+              if (column.name !== 'Initial Review' && column.name !== 'Final Review') {
+                return // no reason to search through other columns and avoids unnecessary API calls
+              }

```

Comment:
I could add logging if neither of 'Initial Review' and 'Final Review' columns are not found. However, that would require extra variables to track that possibility just for the sake of logging. I am leaning towards not adding it to keep things simple, but feel free to object.